extension/oauth2clientauthextension/README.md

```diff
@@ -75,10 +75,16 @@ Following are the configuration fields
 - **client_id_file** - The file path to retrieve the client identifier issued to the client.
   The extension reads this file and updates the client ID used whenever it needs to issue a new token. This enables dynamically changing the client credentials by modifying the file contents when, for example, they need to rotate. <!-- Intended whitespace for compact new line -->  
   This setting takes precedence over `client_id`.
+- **client_id_cmd** - The command to retrieve the client identifier issued to the client. Expects a list of strings.
```

Comment:
Is this the only list of strings we have on this file? If so, please provide an example. Add another example in the testdata's config as well please.

Reply:
We also have the `scopes` field as a list of strings. Nevertheless, I'll push an extra commit to showcase this.